crates/runtime/src/dataconnector/debezium.rs

```diff
@@ -367,14 +379,16 @@ async fn get_metadata_from_kafka(
     let Some(schema_fields) = msg.value().get_schema_fields() else {
         return Err(super::DataConnectorError::UnableToGetReadProvider {
             dataconnector: "debezium".to_string(),
-            source: "Could not get Arrow schema from Debezium message".into(),
+            source: "Could not get Arrow schema from Debezium message".into(), // TODO: what action can a user take from this error?
```

Comment:
I will address this `TODO:` in a future PR as part of #3506

Reply:
This means that the message we got wasn't in a format that we expected for Debezium. That could mean this topic wasn't populated by Debezium or that the Debezium format changed enough that our code is no longer accurate. There isn't much for the user to do, other than validate that the topic was actually the correct one they intended to connect to.